test/e2e/vault-decryption-chrome.spec.js

```diff
@@ -29,6 +29,9 @@ async function getExtensionStorageFilePath(driver) {
     'Default',
     'Local Extension Settings',
   );
+  if (!extensionsStoragePath || !fs.existsSync(extensionsStoragePath)) {
```

Comment:
Is this the case where the test is not run against a dist/prod build? If so, maybe we should throw an error with a useful message, instead of letting the test pass?

Or are you seeing this problem on CI as well?

Reply:
Yeah this is similar to my question. Why would this path be missing?

Reply:
> Is this the case where the test is not run against a dist/prod build? If so, maybe we should throw an error with a useful message, instead of letting the test pass?

That could explain it! Agreed better error message sounds like better approach. It's clear now that "test build" in the comment is referring to the build type, not "in a test setting" which was the source of confusion here.